cmd/eval_test.go

```diff
@@ -1543,3 +1544,150 @@ func TestBundleWithStrictFlag(t *testing.T) {
 	}
 
 }
+
+func TestEvalDiscardOutput(t *testing.T) {
```

Comment:
Suggestion: it'd be nice to have these tests immediately after the other output tests, i.e. `TestEvalPartialFormattedOutput`